web/src/components/tasks/business-formation/getErrorStateForField.test.ts

```diff
@@ -1253,6 +1264,16 @@ describe("getErrorStateForField", () => {
       }
     };
 
-    runTests(hasErrorIfEmpty);
+    runTests(onlyHasErrorIfEmptyTest);
+  });
+
+  describe("confirm field names in test and getErrorStateForField component match", () => {
+    it("onlyHasErrorIfEmpty", () => {
+      expect(onlyHasErrorIfEmpty).toEqual(onlyHasErrorIfEmptyTest);
```

Comment:
pebble - if your importing the arrays, why not reference them and the individual test as opposed to creating an entirely different array, using them in the test, and then having another test to verify that they match the imported arrays. What's the value of the current approach?